pom.xml

```diff
@@ -11,7 +11,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.3.3.RELEASE</version>
+        <version>2.3.7.RELEASE</version>
```

Comment:
Tror denne branch trenger en rebase for å annulere disse endringer. Mener jeg så de i en annen branch.

Reply:
Ja, er fullt klar over det :p